### ISSUE TITLE
reading tedges from a file: error  

### ISSUE THREAD

**Issue by abitofalchemy** [IngoScholtes/pyTempNets#16]
There is an inconsistency between the instructions in the Tutorial linked to on the README.md file and the actual file in the pyTempNet folder: `example.tedges`
- The actual format of the input file is:

```
==> example.tedges <==
time node1 node2
0 29 22
1 22 25
2 24 29
3 29 22
```

With time as the first column.  But in the Tutorial, the instruction is to have the time (property or attribute) follow the edge, which I find more natural.

```
==> ex2.tedges <==
source target time
a b 4
b c 7
u v 3
u v 3
```

If I try to read in either format files, I get the following error:

```
Traceback (most recent call last):
  File "tst.py", line 35, in <module>
    t = tn.readFile('ex2.tedges', sep=' ')
  File "/home/saguinag/.local/lib/python2.7/site-packages/pyTempNet/Utilities.py", line 73, in readFile
    Log.add('Reading time-stamped links ...')
TypeError: unbound method add() must be called with Log instance as first argument (got str instance instead)
```

is the `Log.add` as shown in the error msg just missing something ?

Lastly, is there a reason for not using a character (such as a `#` or `%`) on the header line to denote 'header comments' ?

**Comment by romanc** [comment]
This is another python2 <-> python3 compatibility issue to be fixed for python2 in pull-request #13 of yesterday. Please note, that the project currently officially only supports python3.

For the format of the input files the order of source/target/time is irrelevant. The corresponding rows will be assigned according to the keywords in the header line. This is documented (see docsting of the function `readFile()` in class `Utilities`) behavior.

**Comment by IngoScholtes** [comment]
Thanks, Roman for fixing this!
